Gemfile.lock

```diff
@@ -4,22 +4,22 @@ PATH
     high_voltage (2.4.0)
 
 GEM
-  remote: http://rubygems.org/
+  remote: https://rubygems.org/
   specs:
-    actionpack (4.2.0)
-      actionview (= 4.2.0)
-      activesupport (= 4.2.0)
-      rack (~> 1.6.0)
+    actionpack (4.2.5.2)
```

Comment:
In clearance I've kept Gemfile.lock checked in. It represents a known state that should work for anyone cloning the project.

The gemfile.lock files created by appraisal are *not* checked in, so CI runs against latest allowed (and I can run against latest allowed with `appraisal rake`).